### PR TITLE
docs: remove name field from Mesh config entry

### PR DESCRIPTION
**Pull request by dnephin**
Follow up to #10155

### CODE REVIEW

File: website/content/commands/config/delete.mdx [hunk 1]
```diff
@@ -27,7 +27,9 @@ Usage: `consul config delete [options]`
 
 - `-kind` - Specifies the kind of the config entry to read.
 
-- `-name` - Specifies the name of the config entry to read.
+- `-name` - Specifies the name of the config entry to read. The name of the
```
Comment (jsosulska) [inline website/content/commands/config/delete.mdx:30]:
```suggestion
- `-name` - Specifies the name of the config entry to delete. The name of the
```

**Review by jsosulska — COMMENTED** [review]
Just one comment. I will let Freddy approve, but so far this LGTM!

**Review by rboyer — APPROVED** [review]
LGTM

File: website/content/commands/config/delete.mdx (hunk 1 above)
Comment (dnephin) [reply website/content/commands/config/delete.mdx:30]: Good catch!

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/364562.

**Comment by hc-github-team-consul-core** [comment]
:cherries::x: Cherry pick of commit 74d516dcfdc144f28a0ccecc2d8bc18f019d4246 onto `stable-website` failed! [Build Log](https://circleci.com/gh/hashicorp/consul/364562)

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 74d516dcfdc144f28a0ccecc2d8bc18f019d4246 onto `release/1.10.x` succeeded!

**Pull request merged**